src/main/java/org/wildfly/maven/plugins/licenses/KnownLicenses.java

```diff
@@ -144,6 +148,8 @@ public class KnownLicenses {
         KNOWN_LICENSES.put("epl 2.0", EPL_2_0_LICENSE);
         KNOWN_LICENSES.put("epl-2.0", EPL_2_0_LICENSE);
 
+        KNOWN_LICENSES.put(FSF_ALL_NAME.toLowerCase(), FSF_ALL_LICENSE);
```

Comment:
The others aren't like this, but might be worth considering adding `toLowerCase(Locale.ROOT)`.

Reply:
@jamezp does it matter? I don't expect the license names to contain non-ASCII characters.

Reply:
Probably not. I would assume it wouldn't either TBH, but it's just always safe :)

Reply:
@jamezp I filed https://github.com/wildfly/licenses-plugin/issues/12 as a 'good-first-issue'.